order.js

```
@@ -50,4 +50,8 @@ router.patch("/:id/status", (req, res) => {
   res.json({ data: { id: req.params.id, status, updatedAt: new Date().toISOString() } });
 });
 
+const ORDER_STATUSES = ["pending", "processing", "shipped", "delivered", "cancelled"];
```

Biome (2.4.4)

[error] Shouldn't redeclare 'ORDER_STATUSES'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)